src/constants/endpoint.ts

```diff
@@ -1,9 +1,3 @@
-import urlJoin from 'url-join';
-
 export const BASE_ENDPOINT = '/api';
-export const JWKS_ENDPOINT = '/oauth/jwks';
 
-export const ARRANGER_READY_ENDPOINT = urlJoin(BASE_ENDPOINT, 'ping');
-export const AUTH_ENDPOINT = urlJoin(BASE_ENDPOINT, 'auth');
-export const HEALTH_ENDPOINT = urlJoin(BASE_ENDPOINT, 'health');
-export const STORAGE_ENDPOINT = urlJoin(BASE_ENDPOINT, 'storage');
+export const ARRANGER_READY_ENDPOINT = '/ping';
```

Comment:
Note: This isn't used anywhere, should probably remove.

Reply:
To document what we discussed over slack, this is used by K8s as arranger's own health endpoint.

https://github.com/supercluster-covid-data-portal/infra/blob/main/helm/api/dev/values.yaml#L64